docs/src/man/guide.md

```diff
@@ -61,13 +61,15 @@ a single empty file at the moment, but we'll be adding to it later on.
 Add the following to your `make.jl` file
 
 ```julia
+push!(LOAD_PATH,"../src/")
```

Comment:
Option 2, just needs a note below this code block warning readers that they will need to manually setup their `LOAD_PATH` or use `include`.